python/doc/examples/probabilistic_modeling/python_distribution.ipynb

```diff
@@ -15,7 +15,7 @@
     "\n",
     "The way to go is inheriting the PythonDistribution class and overload the methods of the Distribution object.\n",
     "\n",
-    "To create a Copula, the user has to overload isCopula() and return True.\n",
+    "To **create a Copula, the user has to overload isCopula() and return True**.\n",
```

Comment:
Please remove the unnecessary bold font: the reader has to read the text anyway.

Reply:
I think it is important to highlight this part as `Copula` class do not exist anymore.
Our future objective is to enrich this example later with a python `Copula`.
By the way, I remove for the moment "this highlight"